modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaPlayFrameworkCodegen.java

```diff
@@ -153,6 +153,9 @@ public void processOpts() {
         supportingFiles.add(new SupportingFile("routes.mustache", "conf", "routes"));
 
         //App/Utils folder
+        if (!this.controllerOnly && this.useInterfaces) {
```

Comment:
Could you add an option to disable emitting the Module or to emit the interfaces only? We are integrating swagger-codegen in our SBT build and we would like to only have the interfaces generated.